### PR TITLE
Fix broken link for "Edge Functions -> Connect to Postgres" example 

### PR DESCRIPTION
**Pull request by dsukruth**
## What kind of change does this PR introduce?
This commit fixes the broken link issue #13328. The link was updated to the correct one. 

## What is the current behavior?

issue #13328
image

## What is the new behavior?

image

Add any other context or screenshots.
N/A

### CODE REVIEW

**Review by kiwicopple — APPROVED** [review]

**Review by saltcod — APPROVED** [review]

File: packages/ui/src/components/Command/CommandMenuProvider.tsx [hunk 1]
```diff
@@ -104,6 +104,7 @@ function useKeyboardEvents({
           return
         case 'k':
         case '/':
+          event.preventDefault()
```
Comment (MildTomato) [inline packages/ui/src/components/Command/CommandMenuProvider.tsx:107]: @dsukruth did this sneak in?

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/Command/CommandMenuProvider.tsx (hunk 1 above)
Comment (dsukruth) [reply packages/ui/src/components/Command/CommandMenuProvider.tsx:107]: Yes, It was a fix of #13339 I probably pushed both commits in the same PR.
**preventDefault()** should be called on the event object passed to the event listener function. This tells the browser to cancel the default action for that particular in this case which opens up the chrome search tab if not prevented.

Comment (dsukruth) [reply packages/ui/src/components/Command/CommandMenuProvider.tsx:107]: If you want me to revert it kindly let me know

**Comment by gregnr** [comment]
Just realized we got a dupe on this that was merged in today. Sorry about that @dsukruth - thanks again for contributing!

**Pull request closed without merging**